Automattic/pocket-casts-android
# Onboarding Flow: Feature flag the Continue with Google button

## Description

This change hides the "Continue with Google" single sign on button behind a feature flag. The SSO may be finished after we release the new onboarding, that is why it's a separate flag.
The button is also hidden if the user hasn't installed Google Play Services. 

## Testing Instructions

1. Update the ONBOARDING_ENABLED and SINGLE_SIGN_ON_ENABLED build flags in base.gradle to be "true".
2. Open the app on an emulator with Google Play Services installed.
3. ✅ Verify the "Continue with Google" button is visible.
4. Open the app on an emulator that doesn't have Google Play Services installed.
5. ✅ Verify the "Continue with Google" button is not visible.
6. Update the SINGLE_SIGN_ON_ENABLED build flags in base.gradle to be "false". 
7. Open the app on an emulator with Google Play Services installed.
8. ✅ Verify the "Continue with Google" button is not visible.

## Screenshots or Screencast 

<img  width="300" src="https://user-images.githubusercontent.com/308331/201820838-a6772c41-598a-461e-a0ff-4054899ca35c.png" />
